bindings/python/iota_sdk/wallet/account.py

```diff
@@ -335,7 +335,7 @@ def get_balance(self) -> Balance:
         ))
 
     def prepare_output(self, params: OutputParams,
-                       transaction_options: Optional[TransactionOptions] = None):
+                       transaction_options: Optional[TransactionOptions] = None) -> TreasuryOutput | AliasOutput | BasicOutput | FoundryOutput | NftOutput | Output:
```

Comment:
prepare_output() can only return Basic or NFT outputs, so should be this?
```suggestion
                       transaction_options: Optional[TransactionOptions] = None) -> BasicOutput | NftOutput:
```